src/sentry/eventstore/snuba/backend.py

```diff
@@ -34,13 +31,23 @@ def get_events(
         """
         Get events from Snuba.
         """
+        assert filter, "You must provide a filter"
+
+        filter_keys = {}
+        if filter.project_ids:
+            filter_keys["project_id"] = filter.project_ids
+        if filter.group_ids:
+            filter_keys["group_id"] = filter.group_ids
+        if filter.event_ids:
+            filter_keys["event_id"] = filter.event_ids
```

Comment:
Isn't this a copy paste of the property already present in the Filter class ?

Reply:
Ah thanks, forgot to clean these up